pages/team.ca.md

```diff
@@ -157,9 +157,9 @@ Desenvolupament.
 ## Consell
 
 L'entorn d'escriptori MATE té un consell directiu. La
-[pàgina wiki del consell de MATE](http://wiki.mate-desktop.com/board)
+[pàgina wiki del consell de MATE](https://wiki.mate-desktop.com/board)
 llista els membres actuals i descriu els seus deures.
 
 <small>
-Les banderes que s'utilitzen en aquesta pàgina han estat proporcionades per <http://www.icondrawer.com>.
+Les banderes que s'utilitzen en aquesta pàgina han estat proporcionades per <https://www.icondrawer.com>.
```

Comment:
`http` works

`https` works but it shows a weird look, the look is fine with `http` (tested with firefox and chromium)

Reply:
Agree, back to http. done and forced-pushed.